continuous_integration/environment.yaml

```diff
@@ -2,30 +2,17 @@ name: test-environment
 channels:
   - conda-forge
 dependencies:
-  - xarray
-  - dask
-  - pykdtree
-  - distributed
-  - cartopy
-  - matplotlib
-  - toolz
-  - Cython
   - sphinx
-  - scipy
   - pyyaml
-  - netifaces
```

Comment:
If you install posttroll with pip, I highly recommend installing netifaces with conda!

Reply:
Re-added in https://github.com/pytroll/pytroll-aapp-runner/pull/31/commits/80dbc52c2cbbb82e616f8b8e4d45d1f90f2b369a